src/content/development/shipyard/settings/_index.en.md

```diff
@@ -62,3 +63,18 @@ clusters:
     submariner: false
     nodes: control-plane
 ```
+
+The following settings file deploys two clusters.
+The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
+As no **`gateways`** setting is specified either globally or for the first cluster specifically,
+the first cluster will get have a single gateway node by default.
```

Comment:
```suggestion
As no **`gateways`** setting is specified either globally or for the first cluster specifically,
the first cluster will get have a single gateway node by default.
The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
```

Somehow reading "the second cluster" as the first paragraph, and then "... the first cluster" as the second, feels wrong to me.
Ordering them makes it better IMO